src/main/java/org/lovetropics/peekaboo/client/DisguiseRenderer.java

```diff
@@ -72,6 +73,8 @@ public static void onRenderPlayerPre(RenderLivingEvent.Pre<?, ?, ?> event) {
     public static void onRenderPlayerPost(RenderLivingEvent.Post<?, ?, ?> event) {
         DisguiseRenderState disguiseState = event.getRenderState().getRenderData(DisguiseRenderState.KEY);
         if (disguiseState != null && disguiseState.entityRenderState() == null) {
+            EntityRenderDispatcher dispatcher = Minecraft.getInstance().getEntityRenderDispatcher();
+            ((HasConditionalShadowRendering) (Object) dispatcher).ltMods$restoreOriginalRenderShadows();
```

Comment:
This is in the `entityRenderState == null` branch, probably shouldn't be? :slightly_smiling_face: